lib/matplotlib/tests/test_colors.py

```diff
@@ -555,7 +555,7 @@ def test_PowerNorm():
     assert_array_almost_equal(norm(a), pnorm(a))
 
     a = np.array([-0.5, 0, 2, 4, 8], dtype=float)
-    expected = [0, 0, 1/16, 1/4, 1]
+    expected = [-1/16, 0, 1/16, 1/4, 1]
```

Comment:
What happens to the inverse? On line 563, I see that it is tested, but excludes the value below `vmin`. But that was when it became 0 for everything, which could never be inverted uniquely; is it uniquely invertible now?

Reply:
Ah you're very correct that I didn't change the inverse, I've marked this as draft again so it doesn't get merged until I fix that.

Reply:
Thanks for catching, should be fixed now with an updated test to make sure the inverse is tested.